INSTALL

```diff
@@ -35,29 +35,15 @@ One can have other settings that are not exposed to users in private.properties,
 
 
 *****************How to Install:*****************
+in advance you need mvn install local for azkaban-common(commpile('azkaban:azkaban-common:3.0.0')) and azkaba-web-server(compile('azkaban:azkaban-web-server:3.0.0')).
```

Comment:
Since we are refactoring this already, can we put the installation instruction in README.md. Then we can remove this file.

also spelling error in `azkaban-common(commpile('azkaban:azkaban-common:3.0.0'))`